tests/System.Numerics.Tensors.Tests/TensorSliceTests.cs

```diff
@@ -322,7 +322,6 @@ public void SliceIndexOutOfRange(TensorConstructor constructor)
         {
             var slice = Get3DSlice(constructor);
 
-            Assert.Throws<ArgumentNullException>(() => slice[(int[])null]);
```

Comment:
@eerhardt, please note this change. Is this fine?

Reply:
We pass this arg directly into Span's constructor.  What does Span do now when you give it a null `int[]`?

I think we should keep this test, and add an explicit check for null in the indexer.  No matter what, the test is valuable to ensure the behavior of passing null into this method is as expected.

---
In reply to: [167384904](https://github.com/dotnet/corefxlab/pull/2111#discussion_r167384904) [](ancestors = 167384904)

Reply:
Passing null to Span returns `default(Span<T>)`

Reply:
The test isn't testing passing a null to Span.  The test is testing passing a null to the indexer of Tensor<T>.  It just so happens that the implementation of Tensor<T> is directly passing it into a Span.  But the test doesn't care what it does under the covers.

Reply:
Would it hurt performance too much if we added a null check to the indexer?

I will add back the test case and update the indexer to check for null.